grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitCleanJobRepositoryServlet.groovy

```diff
@@ -0,0 +1,27 @@
+package com.twcable.grabbit.client.servlets
```

Comment:
I think having a new resource (`/grabbit/jobrepository`) may be better to express the behavior. Since, the functionality provided by the Servlet is solely on the `JobRepository`... 

`DELETE /grabbit/jobrepository/12` or `DELETE /grabbit/jobrepository?hours=12` or something...